docs/essentials/pipeline-queues.md

```diff
@@ -0,0 +1,197 @@
+---
+description: See how to assign pipelines to execution queues and run them
+sequentially or how to run them in parallel to save time.
+---
+
+# Pipeline Queues
+
+Pipeline queues allow you to control which pipelines Semaphore must run
+sequentially and which may run in parallel. For example, you may configure
+parallel pipelines on the main branch, while allowing only one deployment to
```

Comment:
For a minute this got me confused by thinking how is this related to parallel blocks. Consider rephrasing, maybe this is a case for using the term workflow instead? Here by "pipelines" I understand you mean "instance of a running pipeline" (aka workflow).

Reply:
Workflow is just a container for pipelines and it has no state on its own, so it is not a good fit here, especially because queuing is configured separately for each pipeline in the workflow. 

I adjusted this slightly, but anything else would IMO just make this intro more complicated and it should be clearer from the rest of the article.